### ISSUE TITLE
Uncaught TypeError: Cannot read property 'path' of undefined

### ISSUE THREAD

**Issue by jerabaul29** [nteract/hydrogen#413]
**Atom Version**: 1.9.9
**System**: Ubuntu 16.04.1
**Thrown From**: [Hydrogen](https://github.com/nteract/hydrogen) package, v0.12.4

### Stack Trace

Uncaught TypeError: Cannot read property 'path' of undefined

```
At /home/hydroubuntu/.atom/packages/Hydrogen/lib/kernel-manager.coffee:72

TypeError: Cannot read property 'path' of undefined
    at KernelManager.module.exports.KernelManager.startKernelFor (/home/hydroubuntu/.atom/packages/Hydrogen/lib/kernel-manager.coffee:60:60)
    at Object.module.exports.Hydrogen.createResultBubble (/home/hydroubuntu/.atom/packages/Hydrogen/lib/main.coffee:204:24)
    at Object.module.exports.Hydrogen.run (/home/hydroubuntu/.atom/packages/Hydrogen/lib/main.coffee:290:14)
    at atom-text-editor.subscriptions.add.atom.commands.add.hydrogen:run (/home/hydroubuntu/.atom/packages/Hydrogen/lib/main.coffee:50:33)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:260:29)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -0:00.2.0 hydrogen:run (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {},
  "Hydrogen": {
    "kernelspec": "{\n  \"kernelspecs\": {\n    \"python2\": {\n      \"spec\": {\n        \"display_name\": \"Python 2\",\n        \"language\": \"python\",\n        \"argv\": [\"python2.7\", \"-m\", \"ipykernel\", \"-f\", \"{connection_file}\"],\n        \"env\": {}\n      }\n    }\n  }\n}"
  }
}
```

### Installed Packages

```coffee
# User
Hydrogen, v0.12.4 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.2 (active)
one-light-ui, v1.3.2 (inactive)
one-dark-syntax, v1.2.0 (active)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.1 (active)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.1 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.98.1 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.2 (active)
language-gfm, v0.86.0 (active)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.22.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.3 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.8 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by jerabaul29** [comment]
I googled the issue. While several other people reported similar problems with older versions of Atom and / or other packages, I could not find an answer to this issue with Hydrogen on a recent version of Atom. I tried the Ubuntu 16.04LTS solution indicated in the package settings (section Debian 8 and Ubuntu 16.04 LTS) but it did not solve it for me (hopes i did it right...).

**Comment by lgeiger** [comment]
Oh I thought we fixed this a while back. I'll fix it tomorrow (for real now 😉 ).

In the meantime you can add any project folder via `File > Add Project Folder` to prevent this error.

**Comment by jerabaul29** [comment]
That sounds great, thanks ;)

**Comment by lgeiger** [comment]
@jerabaul29 We just released `v0.13.0` to fix this.

**Comment by jerabaul29** [comment]
Thanks! I will test it this afternoon.
